packages/sdk/src/index.ts

```diff
@@ -90,3 +90,6 @@ export {
   handleMessageExtensionQueryWithSSO,
 } from "./messageExtension/executeWithSSO";
 export { MessageExtensionTokenResponse } from "./messageExtension/teamsMsgExtTokenResponse";
+
+import * as BotBuilderCloudAdapter from "./conversationWithCloudAdapter/conversationWithCloudAdapter";
```

Comment:
Will we consider to make `BotBuilderCloudAdapter` the default one on next major version release?

Reply:
I propose to remove `BotBuilderCloudAdapter` on next major version release and migrate the changes back to `conversation` from `conversationWithCloudAdapter`.